README.md

```diff
@@ -12,10 +12,11 @@
 
 ## Table of Contents
 
+- [Table of Contents](#table-of-contents)
```

Comment:
I'm not an expert on that but is it useful? Most of the time I don't see the TOC being added to the TOC /cc @lizrice

Reply:
I didn't notice this. I have VS code extension for Markdown and it automatically generates TOC. I can revert this change too if you wish.